fe1-web/components/ProfileIcon.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Blockies from 'react-blockies';
+import { PublicKey } from 'model/objects';
+import { popBlue, popGray } from 'styles/colors';
+
+const ProfileIcon = (props: IPropTypes) => {
```

Comment:
Maybe add a test for this component?

Reply:
I really would have loved to, but as I said in the PR description, the library doesn't work in tests. Since it works fine in the application, I decided to mock them instead of testing them (as we did with icons).